sda-commons-dependencies/build.gradle

```diff
@@ -15,6 +15,8 @@ ext {
   weldVersion = '3.1.4.Final'
   openTracingUtilVersion = '0.33.0'
   scalaVersion = '2.12.11'
+  kafkaVersion = '2.2.1'
+  confluentVersion = '5.2.3'
```

Comment:
The latest version is 5.3.0. Do we want to update this later through dependabot?

Reply:
Let's discuss our strategy for updating dependencies like Kafka / MongoDB / Postgresql. Version 5.2.3 is the current version that is supported by IBM's default plan. We decided to update to this version in the last architectural decision: https://sda-se.atlassian.net/wiki/spaces/platform/pages/849510462/Synchronize+Kafka+Versions